### ISSUE TITLE
Exception in ValueChangeEvent still freezes UI

### ISSUE THREAD

**Issue by LennartKuell** [vaadin/flow#4979]
If an Exception is thrown in a ValueChangeEvent it will still freeze the UI. The fix (#4529) in #801 does not work in this case. Exceptions thrown outside of a ValueChangeEvent will be caught by the custom ErrorHandler, like intended.

The ErrorHandler was set up like this: https://github.com/vaadin/flow/blob/master/flow-tests/test-root-context/src/main/java/com/vaadin/flow/uitest/ui/ErrorHandlingView.java
The ValueChangeEvent was fired with the following code, after adding the TextField to the layout and typing something in the TextField and then pressing Enter:
```java
TextField tfDebug1 = new TextField();
tfDebug1.addValueChangeListener(e -> {
    throw new RuntimeException();
});
```

Tested with Vaadin 12.0.3 and flow-server 1.2.3, 1.2.4 and 1.3.0.alpha1. I also tested this both on Firefox and Chromium on Linux Mint.

**Comment by sergetania** [comment]
I am experience this issue despite the fact that I have both an error view (for navigation errors) and an error handler registered with the Vaadin session. After the error, the user is not routed to an error page and the browser needs to be refreshed.

**Comment by esommars** [comment]
This still breaks in Vaadin 13.0.0

**Comment by denis-anisimov** [comment]
The fix should have been cherry picked to `1.4`  branch and released as a part of `1.4.1`.
But it has not been cherry picked.
As a result the release doesn't contain it.
I will cherry pick it but it's too late for the release anyway.
You may use SNAPSHOT Flow version for now or upcoming `1.4.3` release.

**Comment by sergetania** [comment]
This still breaks in Vaadin 13.0.2! Why is the fix for the issue is being ignored? This is not an insignificant problem. Can this be escalated to be included in the next release, please?

**Comment by pleku** [comment]
@sergetania the fix has been included in `13.0.3` which landed yesterday.

**Comment by sergetania** [comment]
It works for me in 13.0.3. Thank you!
